src/versions/package_version/dotnet.ts

```diff
@@ -40,10 +40,10 @@ export function setPackageVersionDotnet(newVersion: string): void {
   fs.writeFileSync(pathToCsproj, csProjWithNewVersion);
 }
 
-function getCsprojAsObject(filePath: string): any {
+async function getCsprojAsObject(filePath: string): Promise<any> {
   const contents = fs.readFileSync(filePath, { encoding: 'utf8' });
 
-  return JSON.parse(xmlToJson(contents));
+  return await parseStringPromise(contents.toString());
```

Comment:
Dieses `await` könntest du entfernen, dann müsste der Build auch wieder laufen. In diesem Fall macht es ehh keinen Unterschied ob du den Rückgabewert von parseStringPromise erst awaitest und dann returnst oder ob du den Rückgabewert direkt returnst.

```suggestion
  return parseStringPromise(contents.toString());
```

Das async müsstest du dann auch entfernen können. (1/2)